smf/clickhouse/config_replica.xml

```diff
@@ -0,0 +1,82 @@
+<clickhouse>
+    <logger replace="true">
+        <level>trace</level>
+        <console>true</console>
+        <log from_env="CH_LOG"/>
+        <errorlog from_env="CH_ERROR_LOG"/>
+        <size>1000M</size>
+        <count>3</count>
+    </logger>
+    <display_name from_env="CH_REPLICA_DISPLAY_NAME"/>
+    <!-- To allow all use :: -->
+    <listen_host from_env="CH_LISTEN_ADDR"/> 
+    <!-- Default is 8123 -->
+    <http_port replace="true" from_env="CH_LISTEN_PORT"/>
+
+    <interserver_http_host from_env="CH_LISTEN_ADDR"/>
+
+    <path replace="true" from_env="CH_DATASTORE"/>
+    <tmp_path replace="true" from_env="CH_TMP_PATH"/>
+    <user_files_path replace="true" from_env="CH_USER_FILES_PATH"/>
+
+    <user_directories>
+        <local_directory>
+            <!-- Path to folder where users created by SQL commands are stored. -->
+            <path replace="true" from_env="CH_USER_LOCAL_DIR"/>
+        </local_directory>
+    </user_directories>
+
+    <format_schema_path replace="true" from_env="CH_FORMAT_SCHEMA_PATH"/>
+
+    <!-- replication settings -->
+    <macros>
+        <shard>01</shard>
+        <replica from_env="CH_REPLICA_NUMBER"/>
+        <cluster>oximeter_cluster</cluster>
+    </macros>
+
+    <remote_servers replace="true">
+        <oximeter_cluster>
+            <!-- TODO: secret handling TBD -->
```

Comment:
Just checking there's an issue tracking this?

Reply:
There is now :smile: ! https://github.com/oxidecomputer/omicron/issues/3823